packages/app-store/dailyvideo/lib/VideoApiAdapter.ts

```diff
@@ -98,6 +98,44 @@ async function processTranscriptsInBatches(transcriptIds: Array<string>) {
   return allTranscriptsAccessLinks;
 }
 
+export const updateMeetingTokenIfExpired = async ({
+  bookingReferenceId,
+  meetingToken,
+  roomName,
+  exp,
+}: {
+  bookingReferenceId: number;
+  meetingToken: string;
+  roomName: string;
+  exp: number;
+}) => {
+  try {
+    await fetcher(`/meeting-tokens/${meetingToken}`).then(ZGetMeetingTokenResponseSchema.parse);
+  } catch (err) {
```

Comment:
If there is a problem fetching meeting token then update the expiry time and save the new meeting token